app/src/main/AndroidManifest.xml

```diff
@@ -12,7 +12,10 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.NApp"
-        android:usesCleartextTraffic="true"> <!-- TODO Cleartext traffic is enabled to allow for easier development, but should be disabled when deployed to production -->
+        android:usesCleartextTraffic="true">
```

Comment:
Please don't remove the comment, it provides some very important information that may otherwise be overlooked.